core/src/com/mygdx/pirategame/College.java

```diff
@@ -3,12 +3,19 @@
 import com.badlogic.gdx.Gdx;
 import com.badlogic.gdx.graphics.Texture;
 import com.badlogic.gdx.graphics.g2d.Batch;
-import com.badlogic.gdx.physics.box2d.*;
+import com.badlogic.gdx.physics.box2d.BodyDef;
```

Comment:
Optimising imports here - we don't need `*` from `box2d`, only 3 things.... waste of memory